docs/guides/runtime-options-overview.mdx

```diff
@@ -177,9 +189,20 @@ Due to differences in the device compilation process, certain runtime features c
   Can be used with pulse gates, but it might not work when using custom gates.
   </TabItem>
 
+  <TabItem value="Gen3" label="gen3-turbo">
+  Incompatible with:
+  - Dynamic circuits
+  - Fractional gates
+  - Pulse gates
+  - `meas_type=kerneled` or `meas_type=avg_kerneled` (measurement level 1)
+
+  Does not support parameter expressions with arithmetic operations.
```

Comment:
Why is this not a bullet point?

Reply:
I guess those extra notes usually have a 'but', so I can pull that parameter expressions one into the bullets.